Harvest.Core/Domain/Project.cs

```diff
@@ -175,6 +175,7 @@ public class Statuses
             public const string ChangeRequested = "ChangeRequested";
             public const string Completed = "Completed";
             public const string AwaitingCloseout = "AwaitingCloseout";
+            public const string FinalInvoicePending = "FinalInvoicePending";
 
             public static List<string> TypeList = new List<string>
             {
```

Comment:
Need to add your new `FinalInvoicePending` status to this list.  Also, please order them more or less in their active order, so completed should be last with the awaiting closeout and final invoice ones just before.